deploy.yml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: devops-sig-8
+  labels:
+    app: devops-sig-8
+  annotations:
+    reloader.stakater.com/auto: "true"
+    configmap.reloader.stakater.com/reload: "devops-sig-8-configmap"
```

Comment:
This line probably is not necessary, annotation above should auto-detect it according to documentation.